### PR TITLE
Update CSP to allow frame-ancestors for the CMS admin

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: bedrock/settings/__init__.py [hunk 1]
```diff
@@ -263,26 +263,41 @@
...
-    # The first path prefix that matches the request.path.startswith will be used, so order them
-    # from most specific to least.
-    "/cms-admin/images/": CMS_ADMIN_IMAGES_CSP,
-}
-CSP_PATH_OVERRIDES_REPORT_ONLY = {}
+    CSP_PATH_OVERRIDES_REPORT_ONLY = {
+        # Order these from most specific to least specific.
+        "/cms-admin/images/": CMS_ADMIN_IMAGES_CSP_RO,
+        "/cms-admin/pages/": CMS_ADMIN_PAGES_CSP_RO,
+    }
```
Comment (stevejalim) [inline bedrock/settings/__init__.py:300]: Do these _need_ to be added to the RO policy as well for things to work, or is this just for completeness?

**Review by stevejalim — APPROVED** [review]
r+wc - thanks!

File: bedrock/settings/__init__.py (hunk 1 above)
Comment (robhudson) [reply bedrock/settings/__init__.py:300]: For completeness and to avoid the error in the console or being sent to report-uri.

**Comment by robhudson** [comment]
I mentioned in the other thread but should have also included here... this change isn't working for me when I test it in the browser. But I think it's cache related since all the headers I can see look good. I'd like to add unit tests for this but it will take a bit of time to figure out how to make an account to get behind the auth, and how to make pages to get to the edit page.

**Comment by robhudson** [comment]
Closing in favor of upcoming PR

**Pull request closed without merging**